README.md

```diff
@@ -1335,7 +1447,10 @@ This is because Mutation Observers queue a [microtask](https://jakearchibald.com
 
 ## Web Component Differences
 
-Skate implements the [Custom Element spec](http://w3c.github.io/webcomponents/spec/custom/) with a custom API but it does not polyfill the native methods. Since Skate is a custom element library, it does not polyfill [ShadowDOM](http://w3c.github.io/webcomponents/spec/shadow/) or [HTML Imports](http://w3c.github.io/webcomponents/spec/imports/).
+Skate implements the [Custom Element spec](http://w3c.github.io/webcomponents/spec/custom/) with a custom API but it does not polyfill the native methods. Since Skate requires that you BYO your own Shadow DOM polyfill. Some polyfills are:
```

Comment:
Good point. The performance tests show this, though. How could we put this into the bullet point as proof?